index.js

```diff
@@ -124,30 +127,32 @@ function createSeries (opts, archs, platforms) {
       download(combination, function (err, zipPath) {
         if (err) return callback(err)
 
-        var tmpDir = path.join(tempBase, platform + '-' + arch + '-template')
-
-        var operations = [
-          function (cb) {
-            mkdirp(tmpDir, cb)
-          },
-          function (cb) {
-            extract(zipPath, {dir: tmpDir}, cb)
-          }
-        ]
-
         function createApp (comboOpts) {
+          var buildDir = path.join(useTempDir ? tempBase : (opts.out || process.cwd()), platform + '-' + arch + '-template')
```

Comment:
Could you extract the ternary statement into a separate variable + if/else statement? It makes this line easier to read.